doc/netplan-yaml.md

```diff
@@ -2,8 +2,54 @@
 title: "YAML configuration"
 ---
 
+## General configuration structure
+
+The general structure of a Netplan YAML file is shown below.
+
+```yaml
+network:
+  version: NUMBER
+  renderer: STRING
+  bonds: MAPPING
+  bridges: MAPPING
+  ethernets: MAPPING
+  modems: MAPPING
+  nm-devices: MAPPING
+  tunnels: MAPPING
+  vlans: MAPPING
+  vrfs: MAPPING
+  wifis: MAPPING
+```
+
+- **version** (number)
+  > Defines what version of the configuration format is used. The only value supported is `2`. Defaults to `2` is not defined.
```

Comment:
This and the other new quote sections (`> `) does not render correctly in the `netplan.5` man page. (see `make && man _build/doc/netplan.5`). I think we need some newlines around the quote sections.